example_multitenant_update_theme.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+CONFIGURATION_PLAYBOOKS_DIR='/encrypted/ansible/edx/playbooks'
+THEMEX_SCRIPT_DIR='/encrypted/ansible/edx/playbooks/themex_script'
+NEW_THEME_OR_JUST_UPDATE='--no-edx-update-run'
+
+if [ "$1" == "--help" ];
+then
+    echo ""
+    echo "script parameters:"
+    echo "'--help' : show current info"
+    echo "'--update-run' : install new theme"
+    echo "'--no-edx-update-run' : default script behaviour! -> just update threme repo and run update_assets"
+    echo ""
+else
+    $THEMEX_SCRIPT_DIR/update_theme.sh -brhim marvel-yellow-theme-eucalyptus https://github.com/raccoongang/themes_for_themex.io.git all_host_edx $CONFIGURATION_PLAYBOOKS_DIR/hosts $CONFIGURATION_PLAYBOOKS_DIR/server-vars.yml
+fi
```

Comment:
Why https://github.com/raccoongang/themes_for_themex.io.git ? Move it to variables